src/audio/google_rtc_audio_processing.c

```diff
@@ -225,14 +337,36 @@ static struct comp_dev *google_rtc_audio_processing_create(
 	if (!cd->tuning_handler)
 		goto fail;
 
-	cd->state = GoogleRtcAudioProcessingCreate();
+	cd->num_aec_reference_channels = GOOGLE_RTC_AUDIO_PROCESSING_DEFAULT_NUM_AEC_REFERENCE_CHANNELS;
+	cd->num_capture_input_channels = GOOGLE_RTC_AUDIO_PROCESSING_DEFAULT_NUM_CAPTURE_INPUT_CHANNELS;
+	cd->num_capture_output_channels = GOOGLE_RTC_AUDIO_PROCESSING_DEFAULT_NUM_CAPTURE_OUTPUT_CHANNELS;
+	cd->num_frames = GOOGLE_RTC_AUDIO_PROCESSING_PERIOD_SAMPLES_PER_CHANNEL;
+
+	int memory_buffer_size_bytes = get_memory_buffer_size_bytes(cd->num_capture_output_channels);
+	if (memory_buffer_size_bytes > 0) {
```

Comment:
based on the function call, this appears to be always true

Reply:
I think 1 is fine but then it needs to be clear it can be modified, e.g. an optional kconfig